app/assets/javascripts/application.js

```diff
@@ -14,3 +14,106 @@
 //= require jquery_ujs
 
 //= require_tree .
+$(document).ready(function() {
+
+  var x = document.getElementById("demo");
+  var minUpdateUserDistance = 0.01 //km
+  var userId = $("div[style='display:none']").attr('id')
```

Comment:
it'd be much better to access this via an id or class.  What if someone decides to add another div with a style of display none?

Also, you shouldn't have styling in your HTML. :japanese_ogre: